package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edge-currency-monero",
-  "version": "0.4.1",
+  "version": "0.4.2",
```

Comment:
You should not change the version as part of the PR.
There is a convention on how to update the version and you should not worry about that here.